java/src/jmri/jmrit/logixng/actions/ExecuteDelayed.java

```diff
@@ -84,7 +81,7 @@ private String getVariables(SymbolTable symbolTable) {
     /**
      * Get a new timer task.
      */
-    private ProtectedTimerTask getNewTimerTask(ConditionalNG conditionalNG, SymbolTable symbolTable) throws JmriException {
+    private ProtectedTimerTask getNewTimerTask(ConditionalNG conditionalNG, SymbolTable symbolTable, long timerDelay, long timerStart) throws JmriException {
```

Comment:
Not critical, but this might be worth a Javadoc comment as to the difference between timerDelay and timerStart.  (They both sound like "how long to wait to do this" to me)

Reply:
Good point. I have added Javadoc.